site/en/r2/tutorials/keras/basic_classification.ipynb

```diff
@@ -855,7 +855,7 @@
         "num_rows = 5\n",
         "num_cols = 3\n",
         "num_images = num_rows*num_cols\n",
-        "plt.figure(figsize=(2*2*num_cols, 2*num_rows))\n",
+        "plt.figure(figsize=(2*2*num_cols+1, 2*num_rows))\n",
```

Comment:
Is there a reason to increase the width by 1?

Reply:
@yashk2810 One of the plot labels in the grid figure were getting slightly covered by the x tick labels added. I added a little room to make this label completely visible.

Reply:
Can you try using this `plt.tight_layout()` before plt.show() and see if it improves the display?

Reply:
@yashk2810 so, I retested this and here are the results. I think I went with the manual adjustment since it spreads quite a bit with tight_layout. Let me know which you prefer and I can do another PR.
tight_layout:
![image](https://user-images.githubusercontent.com/20404635/62095460-5d4f4e00-b235-11e9-8d0d-6d5eee989d09.png)
manual:
![image](https://user-images.githubusercontent.com/20404635/62095525-8c65bf80-b235-11e9-9e27-885c92c2a9ab.png)

Reply:
The thing with doing it manually is, if you change something, you will need to update the figsize too. plt.tighy_layout() does that for you, so it would be nice to use that :)

>  I can do another PR.
You can modify it in this PR also.

Reply:
Good point. I will update on my PR at lunch. Thanks!

Reply:
@yashk2810 This has been updated.

Reply:
Sweet thanks.

LGTM